package.json

```diff
@@ -1,29 +1,30 @@
 {
   "name": "galoy",
   "scripts": {
-    "tsc-check": "tsc --noEmit -p tsconfig.d.json && tsc --noEmit --skipLibCheck",
-    "eslint-check": "eslint \"{src,test}/**/*.ts\"",
-    "eslint-fix": "eslint \"{src,test}/**/*.ts\" --fix",
-    "prettier-check": "prettier --check .",
-    "prettier-write": "prettier --write .",
-    "build": "tsc -p tsconfig-build.json --skipLibCheck && cp -R src/services/price/protos lib/services/price/ && cp src/graphql/old-schema.graphql lib/graphql/ && tscpaths -p tsconfig.json -s ./src -o ./lib",
-    "start": "yarn build && source ./scripts/export-local.sh && node lib/servers/graphql-old-server.js | pino-pretty -c -l",
-    "trigger": "yarn build && source ./scripts/export-local.sh && node lib/servers/trigger.js | pino-pretty -c -l",
-    "watch": "nodemon -V -e ts,graphql -w ./src -x yarn run start",
-    "watch-trigger": "nodemon -V -e ts,graphql -w ./src -x yarn trigger",
-    "kill-graphql": "kill $(lsof -t -i:4000)",
-    "forward-mongodb": "kubectl port-forward $(kubectl get pods -l app.kubernetes.io/name=mongodb | awk '/mongo/ {print $1}') 27017",
-    "cron": "yarn build && source scripts/export-local.sh && node src/servers/cron.js",
-    "exporter": "yarn build && node lib/servers/exporter.js",
-    "lnd-connection": "source scripts/export-local.sh && echo $MACAROON | base64 -d > ./macaroon && echo $TLS | base64 -d > ./tls.cert && lncli --network regtest --macaroonpath=./macaroon --tlscertpath=./tls.cert --rpcserver=${LNDDNS}:${LNDRPCPORT} getinfo",
-    "lncli": "source scripts/export-local.sh && echo $MACAROON | base64 -d > ./macaroon && echo $TLS | base64 -d > ./tls.cert && lncli --network regtest --macaroonpath=./macaroon --tlscertpath=./tls.cert --rpcserver=${LNDDNS}:${LNDRPCPORT}",
-    "postman": "yarn node test/postman/newman.ts",
-    "bitcoin-cli": "source ./scripts/export-local.sh && kubectl exec bitcoind-0 -- bitcoin-cli -rpcuser=rpcuser -rpcpassword=$BITCOINDRPCPASS -regtest",
-    "update-postman": "curl --location --request GET \"https://api.getpostman.com/collections/4e3fee88-7267-44e1-a316-b81e90b0bd2b\" --header \"X-Api-Key: $POSTMANAPIKEY\" | jq . > test/postman/graphql.postman-collection.json",
-    "test": "export JEST_JUNIT_OUTPUT_DIR=\"./artifacts\" && jest --ci --reporters=default --reporters=jest-junit",
-    "test:unit": ". ./.envrc && LOGLEVEL=warn jest --config ./test/jest-unit.config.js --bail --verbose $TEST | yarn pino-pretty -c -l",
-    "test:integration": ". ./.envrc && LOGLEVEL=warn jest --config ./test/jest-integration.config.js --bail --runInBand --verbose $TEST | yarn pino-pretty -c -l",
-    "build-docs": "npx spectaql spectaql-config.yml -1"
+    "build:docs": "./scripts/build-docs",
+    "build": "./scripts/build-tsc",
+    "code:check": "./scripts/code-check",
+    "deps:clean": "./scripts/deps-clean",
+    "deps:reset": "./scripts/deps-reset",
```

Comment:
Strongly disagree with using yarn to drive tasks that do not depend on nodes.

Reply:
Why?

I removed the deps tasks and included their actions directly in other tasks.